raspadorlegislativo/pipelines.py

```diff
@@ -30,6 +28,6 @@ def post(self, item):
             log.info(response.text)
 
     def serialize(self, item):
-        data = {k: v for k, v in dict(item).items() if k not in self.skip}
+        data = {k: v for k, v in dict(item).items()}
```

Comment:
No need for it. `data = dict(item)` is enough : )

The dict comprehension was useful only to skip/exclude some fields.